luxonis_train/core/tuner.py

```diff
@@ -178,3 +201,22 @@ def _get_trial_params(self, trial: optuna.trial.Trial) -> dict[str, Any]:
                 "No paramteres to tune. Specify them under `tuner.params`."
             )
         return new_params
+
+    def _augs_to_indices(self, aug_names: list[str]) -> list[int]:
+        """Maps augmentation names to indices."""
+        all_augs = self.cfg.trainer.preprocessing.augmentations
+        aug_indices = []
+        for aug_name in aug_names:
+            if aug_name == "Normalize":
+                logger.warn(
+                    f"'{aug_name}' should should be tuned directly by adding '...normalize.active_categorical' to the tuner params, skipping."
+                )
+                continue
+            index = [i for i, a in enumerate(all_augs) if a.name == aug_name]
```

Comment:
Could this be handled more efficiently by getting all names outside the for loop and calling `array.index()` method?